orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/manager.xml.j2

```diff
@@ -16,73 +16,69 @@
-         <name>SSOCookieProvider</name>
-         <enabled>true</enabled>
-      </provider>
-      <provider>
-         <role>identity-assertion</role>
-         <name>HadoopGroupProvider</name>
-         <enabled>true</enabled>
-         <param>
-            <name>CENTRAL_GROUP_CONFIG_PREFIX</name>
-            <value>gateway.group.config.</value>
-         </param>
-      </provider>
-      <provider>
-         <role>authorization</role>
-         <name>AclsAuthz</name>
-         <enabled>true</enabled>
-         <param>
-            <name>knox.acl.mode</name>
-            <value>OR</value>
-         </param>
-         <param>
-            <name>knox.acl</name>
-{%- if ldap.adminGroup is defined and ldap.adminGroup is not none %}
-            <value>KNOX_ADMIN_USERS;{{ ldap.adminGroup }};*</value>
-{%- else %}
-            <value>KNOX_ADMIN_USERS;admin;*</value>
-{%- endif %}
-         </param>
-      </provider>
-   </gateway>
-   <service>
-      <role>KNOX</role>
-   </service>
-   <application>
-      <name>admin-ui</name>
-   </application>
-</topology>
\ No newline at end of file
+    <name>manager</name>
+    <gateway>
+        <provider>
+            <role>webappsec</role>
+            <name>WebAppSec</name>
+            <enabled>true</enabled>
+            <param>
+                <name>csrf.enabled</name>
+                <value>true</value>
+            </param>
+            <param>
+                <name>csrf.customHeader</name>
+                <value>X-XSRF-Header</value>
+            </param>
+            <param>
+                <name>csrf.methodsToIgnore</name>
+                <value>GET,OPTIONS,HEAD</value>
+            </param>
+            <param>
+                <name>xframe.options.enabled</name>
+                <value>true</value>
+            </param>
+            <param>
+                <name>xss.protection.enabled</name>
+                <value>true</value>
+            </param>
+            <param>
+                <name>strict.transport.enabled</name>
+                <value>true</value>
+            </param>
+        </provider>
+        <provider>
+            <role>federation</role>
+            <name>SSOCookieProvider</name>
+            <enabled>true</enabled>
+        </provider>
+        <provider>
+            <role>identity-assertion</role>
+            <name>HadoopGroupProvider</name>
+            <enabled>true</enabled>
+            <param>
+                <name>CENTRAL_GROUP_CONFIG_PREFIX</name>
+                <value>gateway.group.config.</value>
+            </param>
+        </provider>
+        <provider>
+            <role>authorization</role>
+            <name>AclsAuthz</name>
+            <enabled>true</enabled>
+            <param>
+                <name>knox.acl.mode</name>
+                <value>OR</value>
+            </param>
+            <param>
+                <name>knox.acl</name>
+                <value>KNOX_ADMIN_USERS;KNOX_ADMIN_GROUPS;*</value>
```

Comment:
@keyki this is the only change I don't understand. Why is `ldap.adminGroup` needed for CB? 

Since these files are under the `/cm/` folder I wasn't sure if this affects any Ambari deployments. I would assume not and we should rely on the CM Knox gateway-site config for group which is `KNOX_ADMIN_GROUPS`.

Reply:
we don't need it.